Copilot/find instances of cite (#294)
symbiotic-engineering/MDOcean: pubs/applied-ocean-research-model/sections/other-appendices.tex

\label{sec:appendix-additional-hydro}
\subsection{Spar Coefficients}
As \Cref{sec:hydro-other} introduces, because of the importance of the damping plate, the spar hydrodynamic coefficients do utilize other methods besides MEEM.

> We approximate the spar added mass coefficient $A_s$ as a frequency-independent quantity using a formula proposed by \citet{philip_damping_2012} based on the displaced water volume in the spherical projection of the damping plate:

Author: ```suggestion
We approximate the spar added mass coefficient $A_s$ as a frequency-independent quantity using a formula that \citet{philip_damping_2012} proposes based on the displaced water volume in the spherical projection of the damping plate:
```

We next turn our attention to the PTO kinematics and dynamics, to specify the generic impedance matrix $\mathbf{Z}_p$ in \Cref{eq:eom-freq-domain}.
A PTO model must capture the dynamics of any elements that transmit, transduce, or condition power between the rigid-body mechanical energy and the end energy product, in this case electricity.
Relevant dynamics include the inertia, stiffness, static friction, mechanical advantage, losses, and electromagnetic coupling of flywheels, shafts, springs, gears, hydraulic circuits, generators, and power electronics.

> Reference \citet{penalba_review_2016} reviews WEC PTO models and reference~\citep{coe_co-design_2025} synthesizes these into a multiport impedance matching network framework to facilitate control co-design.

Author: ```suggestion
\citet{penalba_review_2016} review WEC PTO models and \citet{coe_co-design_2025} synthesize these into a multiport impedance matching network framework to facilitate control co-design.
```

> Reference \citet{coe_co-design_2025} presents a PTO model in cascade matrix form for a single degree of freedom system. 

Author: ```suggestion
\citet{coe_co-design_2025} present a PTO model in cascade matrix form for a single degree of freedom system. 
```

\label{sec:appendix-multiport}

The intrinsic, powertrain, and Th\'evenin-equivalent representations used throughout \Cref{sec:dynamics} are all instances of multiport circuit models, with effort variables $\vec{e}$ and flow variables $\vec{q}$ at each port.

> Following \citep{reveyrand_multiport_2018}, three matrix forms describe the relationships between port variables: impedance ($\mathbf{Z}$), cascade ($\mathbf{a}$, also called ABCD), and hybrid ($\mathbf{h}$):

Author: ```suggestion
Following \citet{reveyrand_multiport_2018}, three matrix forms describe the relationships between port variables: impedance ($\mathbf{Z}$), cascade ($\mathbf{a}$, also called ABCD), and hybrid ($\mathbf{h}$):
```

This form is used for the PTO kinematics, where it naturally accommodates underactuation.
\end{itemize}

> Conversion between the three forms is accomplished using the formulas in reference \citep{reveyrand_multiport_2018}.

Author: ```suggestion
Conversion between the three forms is accomplished using the formulas proposed by \citet{reveyrand_multiport_2018}.
```

Some port relationships are physically well-defined in one convention but degenerate in another (for example, an ideal current source has no impedance representation but a well-defined hybrid representation) so the convention is chosen to match the physics of each subsystem.

% Verify against \citet{coe_co-design_2025} before submission and edit
% if the convention differs.

> The sign convention adopted here treats both flows as positive when entering the multiport, matching \citep{coe_co-design_2025}.

Author: ```suggestion
The sign convention adopted here treats both flows as positive when entering the multiport, matching the suggestion of \citet{coe_co-design_2025}.
```

$a_{\mu\nu} = \frac{1}{2}(r_{\mu}^2-r_{\nu}^2+d_{\mu\nu}^2)/d_{\mu\nu}$ is the distance from 
the center of the $\mu$th circle to the midpoint of the two intersection points, 
and $1i$ is the imaginary unit.

> The closest-point candidates $\mathcal{C}_{cl}$ match \equationautorefname~4 of \citet{mccabe_force-limited_2024},

Author: ```suggestion
The closest-point candidates $\mathcal{C}_{cl}$ match the solution that \citet[\equationautorefname~4]{mccabe_force-limited_2024} derive.
```

Notably, $\Gamma$ is equal to the reflection coefficient at the generator electrical port, which is commonly used in microwave circuit design to quantify impedance mismatch.
\citet{mccabe_force-limited_2024,coe_co-design_2025} further discuss the reflection coefficient for WECs.

> Similar graphical interpretations of constraints for wave energy converters are presented in the studies \citep{mccabe_force-limited_2024,merigaud_geometrical_2023,bacelli_geometric_2013}, with all three visualizing quadratic constraints as circles on the 2D plane.

Author: ```suggestion
Similar graphical interpretations of constraints for wave energy converters are presented by \citet{mccabe_force-limited_2024,merigaud_geometrical_2023,bacelli_geometric_2013}, with all three visualizing quadratic constraints as circles on the 2D plane.
```

%%%%%%%%%%%%%%%
\subsubsection{Comparison to Other Drag Formulations}
\label{subsec:appendix-drag-others}

> The WEC drag model in the study \citet{quartier_influence_2021} also utilizes describing functions with the relative velocity but groups the drag force into damping and stiffness terms rather than damping and excitation terms.

Author: ```suggestion
\citet{quartier_influence_2021} propose a WEC drag model that also utilizes describing functions with the relative velocity but groups the drag force into damping and stiffness terms rather than damping and excitation terms.
```

In other words, it fails to account for the edge case of $r \rightarrow \infty$ described above, where the drag force is purely an excitation force and cannot be expressed as a damping or stiffness.
Furthermore, the drag ``damping'' term in the grouping of \citet{quartier_influence_2021} includes not just forces with a dissipative effect on the body but also the portion of nonlinear drag excitation forces that are in phase with the body velocity.
This creates the confusing possibility for negative apparent damping coefficients, leading to false concerns about violating energy conservation if erroneously interpreted as a true damping coefficient.
The damping-excitation grouping chosen in \Cref{sec:drag} avoids this confusion and guarantees positive values of the damping coefficient $\mathbf{B}_d$.

Another improvement over \citet{quartier_influence_2021} is that the present model considers the fact that the incident wave velocity, and thus the relative velocity, differs across the width of the WEC in the direction of wave propagation $y$.

> Reference \citet{quartier_influence_2021} more simplistically assumes that the wave velocity at all points equals that at the center of the WEC, which is only accurate when the incident wavelength far exceeds the WEC width.

Author: ```suggestion
\citet{quartier_influence_2021} more simplistically assumes that the wave velocity at all points equals that at the center of the WEC, which is only accurate when the incident wavelength far exceeds the WEC width.
```

> This means that Ref.~\citep{quartier_influence_2021} captures only the Froude-Krylov-like component of the drag excitation, neglecting the diffraction-like component.

Author: ```suggestion
This means that \citet{quartier_influence_2021} captures only the Froude-Krylov-like component of the drag excitation, neglecting the diffraction-like component.
```

> Early MDOcean simulations that used the formulation of \citet{quartier_influence_2021} found this approximation to be unacceptable, producing unstable dynamics (negative drag damping coefficients whose absolute value exceeds the radiation damping) at high frequencies and large wave heights in the operational regime.

Author: ```suggestion
Early MDOcean simulations that used the \citet{quartier_influence_2021} formulation found this approximation to be unacceptable, producing unstable dynamics (negative drag damping coefficients whose absolute value exceeds the radiation damping) at high frequencies and large wave heights in the operational regime.
```

This indicates that the larger error in the 2-DOF-drag-MEEM case is not an error in the 2-DOF dynamic model itself, but in the way that a 2-DOF model amplifies errors in drag and hydrodynamic coefficients due to the importance of the phase of motion between each DOF.
The remainder of this subsection details the four-quadrant breakdown across drag-on/drag-off and identical/different hydrodynamic coefficient configurations.

> A WEC-Sim parallel multiple condition run is performed in accelerator mode for 200s with a 100s ramp time using the ode4 solver and a 10~ms timestep, with the default mass properties and body geometry of the WEC-Sim RM3 tutorial.\footnote{This geometry is not identical to the nominal float design used in the Reference Model Report \citet{RM3} and the rest of this paper, with a shorter spar draft, thicker damping plate, wider float frustum diameter, and assumed deep-water depth regime.

Author: ```suggestion
A WEC-Sim parallel multiple condition run is performed in accelerator mode for 200s with a 100s ramp time using the ode4 solver and a 10~ms timestep, with the default mass properties and body geometry of the WEC-Sim RM3 tutorial.\footnote{This geometry is not identical to the nominal float design used in the Reference Model Report \citep{RM3} and the rest of this paper, with a shorter spar draft, thicker damping plate, wider float frustum diameter, and assumed deep-water depth regime.
```

\begin{figure}[htbp]
    \centering
    \includegraphics[width=1.05\linewidth]{figs/from-matlab/wecsim_rpt_multi_drag_on_meem_on__power_mech_unsat.pdf}

>     \caption{Comparison of mechanical power for RM3 report \citet{RM3} and MDOcean}

Author: ```suggestion
    \caption{Comparison of mechanical power for RM3 report \citep{RM3} and MDOcean}
```

The trapezoidal plates are isosceles trapezoids with smaller base $b_1 = \pi D_{f,in}/12$, larger base $b_2=\pi D_f/12$, and perpendicular height $h_0=(D_f-D_{f,in})/2$ (see \Cref{fig:trapezoid}).
The references consulted do not contain structural results for trapezoidal plates, so geometric intuition is used to scale available solutions.

> For example, reference~\citep{young_roarks_2001} contains models for rectangular plates under perpendicular loading.

Author: ```suggestion
For example, \citep{young_roarks_2001} compile models for rectangular plates under perpendicular loading.
```

\begin{equation}\label{eq:bottom-plate-moment}
M_{bot}=\beta q x_{\text{min}}^2
\end{equation}

> for $\beta$ tabulated as a function of $x_{\text{max}}/x_{\text{min}}$ in reference \citet{timoshenko_theory_1959}, and the bending stress is found by plugging this moment into equation~\eqref{eq:plate-stress}.

Author: ```suggestion
for $\beta$ as a function of $x_{\text{max}}/x_{\text{min}}$ that \citet{timoshenko_theory_1959} tabulates, and the bending stress is found by plugging this moment into equation~\eqref{eq:plate-stress}.
```

        \sigma_{\text{cent}}&=-\beta_2 W/t_{f,\text{top},eq}^2
    \end{aligned}
\end{equation}

> using $\beta_1$ and $\beta_2$ tabulated as a function of $x_{\text{max}}/x_{\text{min}}$ in reference \citep{young_roarks_2001}.

Author: ```suggestion
using $\beta_1$ and $\beta_2$ tabulated as a function of $x_{\text{max}}/x_{\text{min}}$ by \citet{young_roarks_2001}.
```

The bending moment per unit length is then found as $M_{\text{top}}=\sigma_{\text{edge}}t_{f,\text{top},eq}^2/6$.

> However, unlike \Cref{eq:bottom-plate-moment}, it is found that \Cref{eq:top-plate-stress} predicts stresses that do not match well with the finite element results for the nominal RM3 design in references \citep{RM3,previsic_reference_2011}.

Author: ```suggestion
However, unlike \Cref{eq:bottom-plate-moment}, it is found that \Cref{eq:top-plate-stress} predicts stresses that do not match well with the finite element results for the nominal RM3 design \citep{RM3,previsic_reference_2011}.
```

\subsection{Damping Plate}
For an annular plate with free outer radius $a$ and fixed inner radius $b$, the nondimensional plate deflection $\overline{\delta}_{\text{plate}}$ and bending moment $\overline{M}_r$ can be calculated as a function of plate aspect ratio $b/a$ for various load cases.
In this case, the plate is loaded by a distributed load (subscript $dis$) equal to $F_{\text{heave}}$, and a concentrated load (subscript $con$) at four equidistant points along the edge, equal to the tubular support reaction force $F_{\text{tube}}$.

> Formulas for the concentrated loading are given in reference \citet{boedo_corrected_1998}, and formulas for distributed loading are given as case 2L of table 11.2 in reference \citep{young_roarks_2001}.

Author: ```suggestion
Formulas for the concentrated loading are given by \citet{boedo_corrected_1998}, and formulas for distributed loading are given by \citet[case 2L, table 11.2]{young_roarks_2001}.
```

    \right)
\end{equation}

> The critical buckling stress $\sigma_{\text{buckle}}$ is found according to \citep{american_bureau_of_shipping_requirements_2022}.

Author: ```suggestion
The critical buckling stress $\sigma_{\text{buckle}}$ is found with the \citet{american_bureau_of_shipping_requirements_2022} methodology.
```
